index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Fork Me! FCC: Test Suite Template</title>
+  <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+  <div id="main">
+    <div id="title">
+      <h1>Dr. Norman Borlaug</h1>
+    </div>
+    <div id="img-div">
```

Comment:
Great job mentioning this to your student but I suggest that you phrase your comment as a suggestion, be polite, and give a reason for the requested change.

### EXAMPLE
I kindly suggest that you add semantic tags, Like nav and header tags. because a  semantic tag clearly describes its meaning to both the browser and the developer.